app/src/renderer/components/common/TmSessionHardware.vue

```diff
@@ -22,8 +22,16 @@
           value="Connecting..."
           @click.native="setStatus('connect')"
         />
+        <p class="ledger-install">
+          Don't have the Cøsmos Ledger App yet? Install it
+          <a
+            href="https://github.com/cosmos/voyager#ledger-cosmos-app"
```

Comment:
I think this is not a good user experience. Let's add the instructions in the UI as stated in the issue.

Reply:
you mean creating a page/modal to show the instructions?

Reply:
Yes, I think that is better UX